data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -584,19 +584,15 @@
 
         [message]
             speaker=Aleron
-            message= _ "Not more bandits... We have nothing left. Begone!"
+            message= _ "More bandits? Begone from here, we have nothing left to give you."
```

Comment:
"Begone from here" is a redundant and nearly non-existent usage. "Begone" is capable of conveying the desired meaning by itself.